scico/test/test_denoiser.py

```diff
@@ -17,7 +51,7 @@ def setup(self):
 
     def test_shape(self):
         assert bm3d(self.x_gry, 1.0).shape == self.x_gry.shape
-        assert bm3d(self.x_rgb, 1.0).shape == self.x_rgb.shape
+        assert bm3d(self.x_rgb, 1.0, is_rgb=True).shape == self.x_rgb.shape
```

Comment:
This was a bug in the previous bm3d tests?

Reply:
I think so.